examples/conflicts/server.js

```diff
@@ -38,7 +39,7 @@ const customGreetingResolutionStrategy = function(serverData, clientData, baseDa
 const resolvers = {
   Mutation: {
     changeGreeting: async (obj, args, { conflict }, info) => {
-      if (conflict.hasConflict(greeting, args)) {
+      if (conflict.hasConflict(greeting, args, info)) {
```

Comment:
Why we need info there?

Reply:
>> Because we need to add things like operation type (query/mutation) and the operation path (getUser/saveUser) in the Prometheus counter. In order to have them there, we need to pass the resolver info to the conflictListener (aka conflictLogger).
It can definitely be improved but I didn't want to waste time on that as we agreed yesterday